src/Factory.php

```diff
@@ -249,7 +258,7 @@ public function getPharRegistry(): PharRegistry {
         if ($this->registry === null) {
             $this->registry = new PharRegistry(
                 new XmlFile(
-                    $this->getConfig()->getHomeDirectory()->file('/phars.xml'),
+                    $this->getConfig()->getHomeDirectory()->file('/database.xml'),
```

Comment:
I think I like `registry.xml` better. What do you think?

Reply:
The only issue I have with `repository.xml` is how close it is to `repositories.xml`.
The two have a different purpose:
 - store the list of all installed phars for the first
 - list known phars that can be downloaded (and installed) for the second

To be honest, I'm not sure that `database.xml` fit either, it's a bit too abstract.
The more I think about it, the more `installed.xml` seem relevant.

Reply:
I suggested `registry.xml` - not `repository.xml' ;-)